spec/etas/cluster_update_spec.rb

```diff
@@ -0,0 +1,122 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+require "clustering/cluster_holding"
+require "clustering/cluster_ht_item"
+require "etas/cluster_update"
+require "etas/overlap_table_update"
+
+RSpec.describe ETAS::ClusterUpdate do
+  let(:update) { ETAS::OverlapTableUpdate.new }
+  let(:h) { build(:holding, organization: "upenn") }
+  let(:ht) { build(:ht_item, :spm, ocns: [h.ocn], billing_entity: "not_same_as_holding") }
+  let(:ht2) { build(:ht_item, :spm, billing_entity: "not_same_as_holding") }
+
+  before(:each) do |_spec|
+    stub_const("BATCH_SIZE", 10)
```

Comment:
See comment above; I'd really like to get rid of these constants.

Reply:
Constants killed.